test/test_api/test_api.py

```diff
@@ -102,19 +104,24 @@ def test_tabular_classification(openml_id, resampling_strategy, backend):
     ) if 'SUCCESS' in str(run_value.status)]
     assert len(succesful_runs) > 1, [(k, v) for k, v in estimator.run_history.data.items()]
 
+    # Delay the rest of the test as it can take time to store model files
```

Comment:
What do you mean? Could you explain a bit more? Preferably in both the comment and the next change.

Reply:
actually I'll remove this. It is not important